lodash.src.js

```diff
@@ -3592,6 +3594,15 @@
         }
         return fn.apply(thisBinding, args);
       }
+
+      if (isAryExact) {
+        var plist = [];
+        for (var i = 0; i < ary; i++) { plist.push('p' + i); }
+        var params = plist.join(',');
+        var fn = wrapper;
+        eval('wrapper = function(' + params + ') { return fn.apply(this, arguments); }');
```

Comment:
@jdalton ok, I understand. I did this and opted to simply re-wrap the wrapper generated by `createWrapper` and use `apply` rather than `call` with `params`. I felt this simplified the code, but am happy to change it.

I also addressed the other issues & rebased the changes into a single commit.